packages/webapp/pages/helloworld/index.tsx

```diff
@@ -40,7 +40,7 @@ export const getServerSideProps: GetServerSideProps<PageProps> = async ({
   const boot = await getFunnelBootData({
     app: BootApp.Webapp,
     cookies: allCookies,
-    id: id as string,
+    id: (id as string) || 'first-funnel',
```

Comment:
why do you add default here? it should be possible to have it undefined actually

Reply:
This was added for testing purpose. Reverting!